### PR TITLE
BAM-1606: Custom tag workflow documentation

### PR DESCRIPTION
**Pull request by csdev**
- Auction API documentation
- Custom publisher tag tutorial

### CODE REVIEW

**Comment by csdev** [comment]
I still need to add Event API and LinkMate API docs, but this should be a good start. Please check for accuracy.

**Comment by lat2118** [comment]
​Hi Chris,

I can take a look at 5pm when I'm off a call -- sorry for the delayed
response!​

Warm Regards,
Laura Taylor

*Marketing & HR Manager*
335 Madison Avenue, 16F | New York, NY, 10017
(562) 243 - 8474  |
​Narrativ.com
<https://mailtrack.io/trace/link/2008b17a9767d4d3eec73f6475b78e27cc8a7def?url=http%3A%2F%2Fnarrativ.com%2F&userId=1980714&signature=31e687fba25c1ddd>

*​​2017 Digiday Signal Award Winner:*
<https://mailtrack.io/trace/link/0bee0ecf75be2f3a9a251452b8ad805854131ab8?url=https%3A%2F%2Fstatic1.squarespace.com%2Fstatic%2F55db494de4b0f408c8e00f07%2Ft%2F59839176f7e0ab6f61d3385f%2F1501794681117%2FDigiday%2BSignal%2BAwards%2B2017%2B%253A%2BDermstore%2BCase%2BStudy.pdf&userId=1980714&signature=24e3c85b54198088>
Best Native/Content Advertising Platform of the Year

File: docs/tagpublishercustom.rst [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+.. code-block:: none
+
+    8132ac19-109a-466e-8037-540a9bd12798
+
+Your tag will need to generate a new Page Session UUID each time a reader
+lands on a different page/article. Include this value where indicated when
+submitting events on your page. Do not reuse Page Session UUIDs across
+different pages or different users.
+
+.. note:: If you have a single-page application (SPA), generating a new
```
Comment (myfashionhub) [inline docs/tagpublishercustom.rst]: I think the publisher Jstag handles this correctly now. It detects page change based on DOM and URL changes.

File: docs/auction.rst [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+.. list-table::
+   :widths: 10 10 80
+   :header-rows: 1
+
+   * - Query Param
+     - Type
+     - Description
+
+   * - a
+     - int64
```
Comment (c-nichols) [inline docs/auction.rst]: string?

File: docs/auction.rst [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+.. list-table::
+   :widths: 35 10 55
+   :header-rows: 1
+
+   * - Field Name
+     - Type
+     - Description
+
+   * - auction_result . **id**
+     - int64
```
Comment (c-nichols) [inline docs/auction.rst]: string?

**Pull request merged**